src/components/VideoPlayer/index.js

```diff
@@ -131,6 +132,7 @@ export default class VideoPlayer extends PureComponent {
       }
     })
 
+    this.video.on('fullscreenchange', this.handleFullscreenChange)
```

Comment:
In the `componentWillUnmout` method, add the following line to clean the listener:

```javascript
this.video.off('fullscreenchange')
```